### PR TITLE
export etaconfig

### PR DESCRIPTION
**Pull request by gurgunday**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -16,4 +16,4 @@ export { default as compile } from "./compile.js";
 export { default as parse } from "./parse.js";
 export { default as render, renderAsync } from "./render.js";
 export { templates } from "./containers.js";
-export { config, config as defaultConfig, getConfig, configure } from "./config.js";
+export { EtaConfig, config, config as defaultConfig, getConfig, configure } from "./config.js";
```
Comment (nebrelbug) [inline src/index.ts:19]: Could you change this to `export type { EtaConfig }` on a new line? I think that will make it clearer that this is a type export.

**Review by nebrelbug — CHANGES_REQUESTED** [review]

**Comment by nebrelbug** [comment]
Thanks for the PR!

File: src/index.ts (hunk 1 above)
Comment (gurgunday) [reply src/index.ts]: Sure! Fixed.

**Review by nebrelbug — APPROVED** [review]
Looks good! Thanks again.

**Pull request merged**